asm/macros/event.inc

```diff
@@ -987,7 +987,9 @@
 	@ Gives the player a Pokémon of the specified species and level, and allows to customize extra parameters.
 	@ VAR_RESULT will be set to MON_GIVEN_TO_PARTY, MON_GIVEN_TO_PC, or MON_CANT_GIVE depending on the outcome.
 	.macro givemon species:req, level:req, item, ball, nature, abilityNum, gender, hpEv, atkEv, defEv, speedEv, spAtkEv, spDefEv, hpIv, atkIv, defIv, speedIv, spAtkIv, spDefIv, move1, move2, move3, move4, isShiny, ggMaxFactor, teraType
-	callnative ScrCmd_givemon
+	callnative ScrCmd_createmon
```

Comment:
Is it correct that both script commands are named `ScrCmd_createmon`?

Also is there a way to combine both scripts since most of it is the same?

Reply:
Both just call the same function which I edited to handle the addition inputs.

As far as using the same macros.. I don't think so since most are optional, but I could be wrong

Reply:
> Both just call the same function which I edited to handle the addition inputs.

Right. 

Though for some reason the CI fails for tests without having run any tests

Reply:
> > Both just call the same function which I edited to handle the addition inputs.
> 
> Right.
> 
> Though for some reason the CI fails for tests without having run any tests

yeah I don't get that... its saying `PARTY_SIZE` is undefined, but global.h is included which includes constants/global.h where that is defined...

Reply:
I would do that unless someone has an idea on what's going on